docs/platforms/javascript/guides/angular/features/component-tracking.mdx

```diff
@@ -196,23 +195,18 @@ import { AppModule } from "./app/app.module";
 
 // ...
 
-const activeTransaction = Sentry.getActiveTransaction();
-const bootstrapSpan =
-  activeTransaction &&
-  activeTransaction.startChild({
+startSpan(
+  {
     description: "platform-browser-dynamic",
```

Comment:
We need to replace description with name. Also, we might wanna change to the suggestion below because `platformBrowserDynamic` isn't used in Angular 17 anymore. 

```suggestion
    name: "bootstrap-angular-application",
```